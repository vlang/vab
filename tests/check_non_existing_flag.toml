execute = 'vab --non-existing-flag'
[expect]
	exit_code = 1
