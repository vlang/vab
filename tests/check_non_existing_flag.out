Error while parsing `os.args`: flag.UnkownFlagError: Unknown flag `--non-existing-flag`
Use `vab -h` to see all flags